mlsshared.sty

```diff
@@ -185,7 +188,7 @@
   breaklines=true,                 % automatic line breaking only at white-space
   keepspaces,                      % don't remove space such as those after closing parenthesis
   captionpos=b,                    % sets the caption-position to bottom
-  commentstyle=\color{commentcolor}\sffamily, % comment style
+  commentstyle=\color{commentcolor}\sffamily\small, % comment style, need smaller size for sans serif font in LaTeXML
```

Comment:
I believe the font size selection here should be set based on the pdfLaTeX appearance, and that we should control the comment font size in the LaTeXML build via the `ltx_lst_comment` CSS class.

Reply:
I would prefer to have it like this for the time being, and then have a discussion whether using sans-serif for comments is really a good idea.

Reply:
OK.  Maybe we could just make the comment even more clear?
```suggestion
  commentstyle=\color{commentcolor}\sffamily\small, % For the LaTeXML build this is smaller than for 'basicstyle', need due to sans serif font.
```